attributes/network/9_account.adoc

```diff
@@ -29,7 +29,8 @@ Some example of dripping wallet's use cases include;
 *   automatic transfer of funds to an execution network as gas payment.
 
 == Requirements
-*   Every account MUST have at least one owner.
+*   An account MUST own itself.
+*   Every account MUST have at least one owner. 
```

Comment:
You can remove this line or clarify that it is impossible to remove an account from owning itself.